### PR TITLE
MPT-31 make sure service is ok at container creation time

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

File: mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/host/Docker.java [hunk 1]
```diff
@@ -57,9 +63,27 @@ public Docker(String imageName)  {
     public ContainerCreation start() throws Exception {
         ContainerCreation container = dockerClient.createContainer(containerConfig);
         dockerClient.startContainer(container.id(), ALL_PORTS_HOST_CONFIG);
+        waitingForSocketToBeReady(container);
         return container;
     }
 
+    private void waitingForSocketToBeReady(final ContainerCreation container) {
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/host/Docker.java:70]: Gorgeous !

**Comment by aduprat** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
:+1:

**Pull request closed without merging**